leaf-ui/js/backendComm.js

```diff
@@ -166,16 +167,14 @@ function getFileResults(isGetNextSteps){
                     console.log("in backendcomm, saving all next state results");
 					open_analysis_viewer();
 				}else{
+					analysisResult = convertToAnalysisResult(results);
                     savedAnalysisData.singlePathResult = results;
-                    analysisResult.assignedEpoch = results.assignedEpoch;
-                    analysisResult.timePointPath = results.timePointPath;
-                    analysisResult.timePointPathSize = results.timePointPathSize;
-                    analysisResult.elementList = results.elementList;
-                    analysisResult.allSolution = results.allSolution;
                     analysisRequest.previousAnalysis = analysisResult;
                     console.log("previousAnalysis");
 					console.log(analysisRequest.previousAnalysis);
-					displayAnalysis(results, false);
+					//pass in an AnalysisResult object
+					
+					displayAnalysis(analysisResult, false);
```

Comment:
Can you make this comment directly above the line it is commenting on? aka remove extra space between lines